.github/workflows/test.yml

```diff
@@ -58,23 +58,23 @@ jobs:
       node-version: ${{matrix.node-version}}
 
   # These tests run with custom docker image attributes that can't be specified in a GHA service
-  database-compose-tests:
+  database-tests-compose:
     needs: build
     strategy:
       fail-fast: false
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        node-version: [16, 18, 20] #, 22]
```

Comment:
Did you mean to stop testing these databases on Node 16 in this PR?

Reply:
Oracle was already disabled because of a timeout.

I also don't see a point to run the tests for HANA under so many node versions.

Reply:
Good point. Seems fine then